README.md

```diff
@@ -79,14 +79,13 @@ this one. The device libraries are available in:
 
 https://github.com/tillitis/tkey-libs
 
-Clone them next this repo and build them first.
+Clone them to the directory above this repo and build them first.
 
 ### Building with Podman
 
-We provide an OCI image with all tools you can use to build the
-tkey-libs and the apps. If you have `make` and Podman installed you
-can us it like this in the `tkey-libs` directory and then this
-directory:
+We provide an OCI image with all the tools needed to build the
+tkey-libs and apps. If you have `make` and Podman installed, you
+can us it for `tkey-libs` directory this directory as shown below:
```

Comment:
"Can us" should be "can use".

"directory this directory" => strike last "this directory"?